bin/oneoff/diff_email_previews

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env ruby
+require 'open-uri'
+require 'fileutils'
+
+# SUMMARY:
+# Comparing content of all workshop email previews before and after a change to make sure the change doesn't have unexpected results.
+
+# List of email previews are copied from http://localhost-studio.code.org:3000/rails/mailers/pd/workshop_mailer/
```

Comment:
Can you generate this list the same way that the page does, so that we don't have to maintain this list here?

Reply:
+1

Reply:
I thought about it but didn't come up with a good solution yet. Will think about it a bit more and update this script with my other email tasks this sprint,

Reply:
If the script runs in dashboard context, rather than having it hit the dashboard web server, it could just use ActiveRecord queries to do all this work like the dashboard already does, couldn't it?